worker/transfer.go

```diff
@@ -100,9 +100,9 @@ func uploadSlab(ctx context.Context, r io.Reader, m, n uint8, hosts []sectorStor
 
 	s.Shards, err = parallelUploadSlab(ctx, shards, hosts)
 	if err != nil {
-		return object.Slab{}, err
+		return object.Slab{}, 0, err
 	}
-	return s, nil
+	return s, length, nil
```

Comment:
Any reason you are returning the `length` here? Seems like we're never using it.

Reply:
it's used in `objectsKeyHandlerPUT`.
It's not *strictly* necessary -- we could wrap the `io.Reader` in a counter to determine the total length of the object -- but this is easier.